examples/SerialIface/vgm.py

```diff
@@ -42,6 +42,11 @@ def play(opl, vgm_stream):
       delay_us = _samples_to_us(delay_samples)
     elif opcode == END_DATA:
       break
+    elif opcode >= 0x51 and opcode <= 0x5f:
```

Comment:
How about failing with `InvalidFormatError()`, but with more information? For example, specify in the message that the command applies to a different YMF chip?

Reply:
OK, I have updated the PR for this.

Reply:
IMO throwing an exception and quiting is too harsh, the file is still perfectly valid. Keep in mind that the VGM format can target multiple chips in one file. For arcade game rips this is more common. Only if the YM3812 clock speed is set to 0 in the header it means that there is no OPL2 data in the file and there is no sense in playing it. In other cases I think it should just ignore the opcodes of different chips, so the parts for the YM3812 can still be played.

For reference the VGM file format spec can be found here: http://www.smspower.org/uploads/Music/vgmspec170.txt

Reply:
I have reverted back to skip the unknown opcodes.